integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -278,6 +278,36 @@ void testAlterTableSetAndRemoveProperty() {
     Assertions.assertTrue(newProperties.containsKey("key2"));
   }
 
+  @Test
+  void testAlterTableAddAndDeleteColumn() {
+    String tableName = "test_column";
+    dropTableIfExists(tableName);
+
+    createSimpleTable(tableName);
+    checkTableColumns(tableName, getSimpleTableColumn(), getTableInfo(tableName));
+
+    sql(String.format("ALTER TABLE %S ADD COLUMNS (col1 string)", tableName));
+    List<SparkColumnInfo> addColumn =
+        Arrays.asList(
+            SparkColumnInfo.of("id", DataTypes.IntegerType, "id comment"),
+            SparkColumnInfo.of("name", DataTypes.StringType, ""),
+            SparkColumnInfo.of("age", DataTypes.IntegerType, null),
+            SparkColumnInfo.of("col1", DataTypes.StringType, null));
+    checkTableColumns(tableName, addColumn, getTableInfo(tableName));
+
+    sql(String.format("ALTER TABLE %S DROP COLUMNS (col1)", tableName));
+    checkTableColumns(tableName, getSimpleTableColumn(), getTableInfo(tableName));
+  }
+
+  private void checkTableColumns(
```

Comment:
seems no need to use a sperate function

Reply:
`a sperate function` is to reduce redundant code.

Reply:
ok